sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -334,6 +334,35 @@ class HiveSparkSubmitSuite
       unusedJar.toString)
     runSparkSubmit(argsForShowTables)
   }
+
+  test("SPARK-29254: include jars passed in through --jars when isolatedLoader is enabled") {
+    def getJarFromUrl(urlString: String): File = {
+      val fileName = urlString.split("/").last
+      Utils.doFetchFile(urlString, Utils.createTempDir(), fileName, new SparkConf, null, null)
+    }
+    val repository = "https://repository.apache.org/content/repositories/releases/"
+    val unusedJar = TestUtils.createJarWithClasses(Seq.empty)
+    val jar1 = TestUtils.createJarWithClasses(Seq("SparkSubmitClassA"))
+    val jar2 = TestUtils.createJarWithClasses(Seq("SparkSubmitClassB"))
+    // download Hive 2.1.1, a non builtinHiveVersion(1.2.1) version for testing
+    val jar3 = getJarFromUrl(s"${repository}org/apache/hive/hive-contrib/" +
```

Comment:
Thank you @LantaoJin Please use our new test API: `HiveTestJars. getHiveContribJar("2.1.1")` and `HiveTestJars. getHiveHcatalogCoreJar("2.1.1")` .

Reply:
Sure